pkg/kubelet/server/stats/fs_resource_analyzer.go

```diff
@@ -29,26 +29,26 @@ import (
 
 type statCache map[types.UID]*volumeStatCalculator
 
-// fsResourceAnalyzerInterface is for embedding fs functions into ResourceAnalyzer
-type fsResourceAnalyzerInterface interface {
+// FileSystemResourceAnalyzer is for embedding fs functions into ResourceAnalyzer
```

Comment:
Could you share more about the rationale for `FileSystemResourceAnalyzer` becoming exported?

Reply:
Because `FileSystemResourceAnalyzer` is an interface needed by other packages (pkg/kubelet/stats).